src/main/java/de/tum/in/www1/artemis/service/connectors/localci/buildagent/LocalCIDockerService.java

```diff
@@ -53,27 +56,55 @@ public class LocalCIDockerService {
     @Value("${artemis.continuous-integration.build-container-prefix:local-ci-}")
     private String buildContainerPrefix;
 
+    @Value("${artemis.continuous-integration.container-cleanup.expiry-minutes:5}")
+    private int containerExpiryMinutes;
+
+    @Value("${artemis.continuous-integration.container-cleanup.cleanup-schedule-minutes:60}")
+    private int containerCleanupScheduleHour;
```

Comment:
this seems to be wrong. the yaml specifies minutes, but the attribute is specified as ScheduleHour